src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/OtherParty.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder(toBuilder = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class OtherParty {
```

Comment:
To stop creating many of these classes, I have introduced a generic one CcdValue<T>. Maybe it's worth taking a look at?